src/docs/product/session-replay/getting-started.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: Set Up
+sidebar_order: 1
+redirect_from:
+  - /session-replay/setup/
+  - /session-replay/getting-started/
```

Comment:
To replace: https://github.com/getsentry/sentry-docs/blob/7197fd3dea0a7ffc92a0d283ebd147ad76356547/src/platforms/javascript/common/configuration/integrations/rrweb.mdx?plain=1#L6

Would require removing this page from the docs on this PR: https://github.com/getsentry/sentry-docs/blob/7197fd3dea0a7ffc92a0d283ebd147ad76356547/src/platforms/javascript/common/configuration/integrations/rrweb.mdx?plain=1#L6

`/platforms/javascript/configuration/integrations/rrweb/` being the current path
and `/platforms/javascript/integrations/rrweb/` the old one
```suggestion
  - /session-replay/getting-started/
  - /platforms/javascript/integrations/rrweb/
  - /platforms/javascript/configuration/integrations/rrweb/
```